src/tools/rust-analyzer/crates/ide-assists/src/tests/sourcegen.rs

```diff
@@ -1,9 +1,12 @@
 //! Generates `assists.md` documentation.
 
+#[cfg(not(feature = "in-rust-tree"))]
```

Comment:
Am I missing something here or why can we not attach the cfg to the `mod` declaration of this `sourcegen` module? Similarly for the `ide-diagnostics` one and the `slow-tests` one. That is I don't see where these differ from the `syntax` crate sourcegen module.

Reply:
Oh let me try that - I think I just assumed those were [auto-discovered](https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html) - but they're actually in `src`, not `tests`!

Reply:
Ah okay, ye the sourcegen test in `slow-tests` in the `rust-analyzer` crate are not in the `src` directory, but in `tests`, so that wouldn't really work, but `ide-diagnostics` and `ide-assists` should be good.

Reply:
Done in 4962241fd7f75055928fba3cbc4bb817a31266f0